app/views/reports/regions/_diabetes_footnotes.html.erb

```diff
@@ -45,7 +45,7 @@
       </div>
       <div class="mb-24px">
         <p class="mb-8px fs-16px fw-bold c-grey-dark">
-          BP <200
+          Blood Sugar &lt;200
```

Comment:
Tiny thing @jamiecarter7 but we generally use "Sentence case" capitalization... so the "Sugar" in each of these should be "sugar"

Reply:
I will correct to sentence case. @dburka